packages/auth-ts/src/core/persistence/react_local.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Persistence,
+  PersistenceType,
+  PersistenceValue,
+  Instantiator
+} from '../persistence';
+
+import AsyncStorage from '@react-native-community/async-storage';
+
+class ReactNativeLocalPersistence implements Persistence {
+  type: PersistenceType = PersistenceType.LOCAL;
+
+  async isAvailable(): Promise<boolean> {
+    return true;
```

Comment:
this should check if actually running in react native

Reply:
We dependency-inject this in for builds of react-native. If this file exists then it is running in react native. Would you me to make an additional check via the helper we've created?

Reply:
for non react-native builds this function will still be available through the legacy polyfill, in those cases it should return false

Reply:
So I suck at closure: Are you saying that `fireauth.storage.AsyncStorage.prototype.get` was part of the public API for the SDK?